packages/manager/modules/exchange/src/account/account.constants.js

```diff
@@ -6,19 +6,51 @@ export const ACCOUNT_COUNTRY_RULE_NAME = 'country';
 
 export const ACCOUNT_WORLD_PHONE_REGEX = /^00\d{2,3}[\s\d]+$/;
 
-export const escapeDescription = (description) =>
-  typeof description === 'string' && description.length > 0
-    ? `"${description}"`
-    : null;
-
+/**
+ * Legacy
+ * @see MANAGER-8716
+ */
 export const unescapeDescription = (description) =>
   typeof description === 'string' ? description.replace(/^"+|"+$/g, '') : null;
 
+const accountPropertiesWithUnit = ['totalQuota', 'usedQuota'];
+const accountPropertiesWithStatus = ['mfa'];
+
+const accountValuetoCSVString = (value) => {
+  const toStringMap = {
+    String: (v) => v,
+    Number: (v) => `${v}`,
+    Boolean: (v) => `${v}`,
+    Undefined: () => '',
+    Null: () => '',
+    Object: (v) => toStringMap.Array(Object.values(v)),
+    Array: (v) => v.map(accountValuetoCSVString).join(','),
+    Date: (v) => v.toISOString(),
+  };
+  const type = Object.prototype.toString.call(value).match(/ (.+?)\]$/)[1];
```

Comment:
`typeof` is not suiting the needs here ?

Reply:
Nope, typeof is not precise enough for some of the types (Array, Date and null will ouput "object")
Object.prototype.toString is more reliable as it gives the native object / primitive in the form of [object Xxxx]